fxpubsub/go.mod

```diff
@@ -0,0 +1,96 @@
+module github.com/ankorstore/yokai-contrib/module/pubsub
+
+go 1.22.0
```

Comment:
is it possible to make it with `go1.20` ? yokai is backward compatible to this version, would be nice if the module follows